observation/interferometric_pointing.py

```diff
@@ -23,9 +23,9 @@ class NoTargetsUpError(Exception):
                   help='Maximum duration of the script in seconds, after which script will end '
                        'as soon as the current track finishes (no limit by default)')
 parser.add_option('--max-extent', type='float', default=1.0,
-                  help='Maximum extent in degrees, the script will scan ')
+                  help='Maximum radial offset from target in degrees, the script will scan ')
```

Comment:
Agreed on this. The original `raster_scan` script has a `--scan-extent` option that is the full length of the scan. Something like `--max-offset` would have made more sense. If this was my script I would just fix the option name, but I'm guessing folks are not keen on changing their SBs...

Reply:
It could  be changed to --max-offset as the current parameters being used are horizon=20, -t 20, and  -m 9000.